### DISCUSSION TITLE
Fast Data Console configurations

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#228] [Q&A] [answered]
### Which section?

Fast Data

### Which is the version?

10.7.1

### Description

Hi!

I Have noticed that when I create Fast Data projections and Single Views, the projection changes collections are added in the  "collections"  array of the "api-console-config.json" file, whereas, the single views and projections are not - they are only added in the "fastdata-config.json" file.

I was wondering if this is the wanted behaviour and whether it is going to change in following releases putting all the fast data configurations in a single file.

Thanks!

**Comment by ianFar96** [comment, marked as answer]
Hi @Lucascanna 

I remember this was introduced mainly to give access to the user to the Projection Changes collection since from the Fast Data section you cannot modify its data model or indexes (while for the Single Views and Projections you can).

I don't think we will migrate Projection Changes collections into the fast-data-config.json since our goal is to migrate from Mongo Projection Changes to Kafka, so hopefully in the future we won't need the Projection changes collections anymore.

**Reply by lucascanna** [reply]
Hi @ianFar96,

Thanks for the precise answer and for claryifing the vision of Fast Data with regards to projection changes.
